pkg/controllers/user/rbac/namespace_handler.go

```diff
@@ -134,23 +143,135 @@ func (n *nsLifecycle) assignToInitialProject(ns *v1.Namespace) error {
 	return nil
 }
 
-func (n *nsLifecycle) ensurePRTBAddToNamespace(ns *v1.Namespace) (bool, error) {
+// cleanRBFromNamespace cleans all role bindings when namespace is moved
+func (n *nsLifecycle) cleanRBFromNamespace(ns *v1.Namespace) error {
 	// Get project that contain this namespace
```

Comment:
I'm a bit concerned with this method. Why do we need it? why didn't we have it before?